applications/MeshingApplication/tests/mmg_lagrangian_test/beam2D_test_result.mesh

```diff
@@ -0,0 +1,952 @@
+MeshVersionFormatted 2
```

Comment:
you sure that this file should be included? it looks like it is a result file...